### ISSUE TITLE
Management Zone Creation not possible for rules with "EXIST" operator

### ISSUE THREAD

**Issue by ARZ-SagA** [dynatrace-oss/terraform-provider-dynatrace#10]
Hi!

Since 1.0.5 it's not possible to create Management Zones with a rule where the Operator "EXISTS"
Example TF Config
```resource "dynatrace_management_zone" "exists-test-mmz" {
	name = "TF - test for bug"
	rules {
		conditions {
			string_comparison {
				type = "STRING"
				negate = false
				operator = "EXISTS"
				value = ""
			}
			base_condition_key {
				attribute = "KUBERNETES_CLUSTER_NAME"
			}
		}
		enabled = true
		type = "KUBERNETES_CLUSTER"
	}
}
```

Output:

```Error: {
  "code": 400,
  "message": "Constraints violated.",
  "constraintViolations": [
    {
      "parameterLocation": "PAYLOAD_BODY",
      "location": "",
      "message": "caseSensitive must be null if the operator 'EXISTS' is specified, for any other operator it must be set.",
      "path": "rules[0].conditions[0].comparisonInfo.caseSensitive"
    }
  ]
}
```
Best regard,

Alex

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Thanks Alex.
We had similar issues with other resources and the same attribute. We will introduce a workaround with the next release.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Fixed with [v1.0.6](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/edit/v1.0.6)
